tests/layer_tests/tensorflow_tests/test_tf_Conj.py

```diff
@@ -0,0 +1,119 @@
+
+        return inputs_data
+
+    def create_complex_conjugate_net(self, input_shape, perm):
+        """
+            TensorFlow net                                 IR net
+            Placeholder->Conjugate    =>       Placeholder->Conjugate
+        """
+
+        tf.compat.v1.reset_default_graph()
+
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            real_part = tf.compat.v1.placeholder(np.float32, input_shape, 'real_part')
+            imag_part = tf.compat.v1.placeholder(np.float32, input_shape, 'imag_part')
+
+            complex_input = tf.raw_ops.Complex(real=real_part, imag=imag_part)
+
+            conj= tf.raw_ops.Conj(x=complex_input, perm=perm, name = "Operation")
+            real = tf.raw_ops.Real(input=conj)
+            img = tf.raw_ops.Imag(input=conj)
+
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        ref_net = None
+
+        return tf_net, ref_net
+
+
+    test_data = [
+        (dict(input_shape=[1, 2], perm=[1, 0])),
+        (dict(input_shape=[1, 2, 3], perm=[2, 1, 0])),
+        (dict(input_shape=[1, 2, 3, 4], perm=[0, 3, 2, 1])),
+        (dict(input_shape=[1, 2, 3, 4, 5, 6], perm=[0, 2, 1, 3, 4, 5])),
+    ]
+
+    @pytest.mark.parametrize("params", test_data)
+    @pytest.mark.precommit_tf_fe
+    @pytest.mark.nightly
+    def test_conjugate(self, params, ie_device, precision, ir_version, temp_dir,
+                                 use_new_frontend):
+        self._test(*self.create_complex_conjugate_net(**params),
+                   ie_device, precision, ir_version, temp_dir=temp_dir,
+                   use_new_frontend=use_new_frontend)
+
+
+class TestConjugate(CommonTFLayerTest):
+
+    def _prepare_input(self, inputs_info):
+
+        assert 'input' in inputs_info
+        input_shape = inputs_info['input']
+
+        inputs_data = {}
+        inputs_data['input'] = np.random.default_rng().random(input_shape).astype(np.float32)
+
+        return inputs_data
+
+    def create_conjugate_net(self, input_shape, perm):
+        """
+            TensorFlow net                                 IR net
+            Placeholder->Conjugate    =>       Placeholder->Conjugate
+        """
+
+        tf.compat.v1.reset_default_graph()
+
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            input = tf.compat.v1.placeholder(np.float32, input_shape, 'input')
+
+            tf.raw_ops.Conj(x=input, perm=perm, name = "Operation")
+
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        ref_net = None
+
+        return tf_net, ref_net
+
+    test_data = [
+        (dict(input_shape=[1, 2], perm=[1, 0])),
+        (dict(input_shape=[1, 2, 3], perm=[2, 1, 0])),
+        (dict(input_shape=[1, 2, 3, 4], perm=[0, 3, 2, 1])),
+        (dict(input_shape=[1, 2, 3, 4, 5, 6], perm=[0, 2, 1, 3, 4, 5])),
+    ]
```

Comment:
`perm` argument is not needed for `Conj` operation. Also, `Conj` has only `input` and `name` key arguments.

Reply:
> `perm` argument is not needed for `Conj` operation. Also, `tf.raw_ops.Conj` has only `input` and `name` key arguments.

The arguments still need to be passed as a dict, right?

Reply:
Use `parameterize` fixture in pytest and leave only `input_shape` parameter. `perm` should be removed. Example of `parameterize` fixture you can find here: https://github.com/openvinotoolkit/openvino/blob/master/tests/layer_tests/tensorflow_tests/test_tf_AddTypes.py